src/index.html

```diff
@@ -1,36 +1,38 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Promise function</title>
-    <link rel="stylesheet" href="./styles/main.scss">
-  </head>
-  <body>
-    <form>
-      <label for="login">
-        Login
-        <input
-          name="login"
-          type="text"
-          placeholder="Login"
-          id="login"
-        >
-      </label>
-      <label for="password">
-        Password
-        <input
-          name="password"
-          type="password"
-          placeholder="Password"
-          id="password"
-         >
-      </label>
+    <head>
```

Comment:
fix it 🥲
<img width="545" alt="image" src="https://user-images.githubusercontent.com/86183007/195558073-a17e4c41-2476-4229-9c8d-1073e7ea5c39.png">